### PR TITLE
PS-5429: encrypt_binlog - remove

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

**Review by laurynas-biveinis — APPROVED** [review]

File: doc/source/.res/text/encrypt_binlog.removing.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Starting from release 8.0.14, |Percona Server| uses the upstream implementation of binary log
+encryption. The variable |encrypt-binlog| is removed and the related command
```
Comment (hrvojem) [inline doc/source/.res/text/encrypt_binlog.removing.txt:2]: should be 8.0.15, as we didn't release 8.0.14.

**Review by hrvojem — CHANGES_REQUESTED** [review]

File: doc/source/management/data_at_rest_encryption.rst [hunk 2]
```diff
@@ -417,35 +412,14 @@ Dumping of encrypted binary logs involves decryption, and can be done using
...
-
-  :cli: ``--encrypt-binlog``
-  :dyn: No
-  :scope: Global
-  :vartype: Boolean
-  :default: ``OFF``
-
-The variable turns on binary and relay logs encryption.
-
 Temporary file encryption
```
Comment (hrvojem) [inline doc/source/management/data_at_rest_encryption.rst:438]: we should leave the description, just add in the :version-info: that it was removed in 8.0.15.

Comment (borys-belinsky) [reply doc/source/management/data_at_rest_encryption.rst:438]: Ready for review

**Review by laurynas-biveinis — APPROVED** [review]

File: doc/source/management/data_at_rest_encryption.rst (hunk 2 above)
Comment (hrvojem) [reply doc/source/management/data_at_rest_encryption.rst:438]: why not link to :rn:`8.0.15-5`" as that would link to the release notes?

File: doc/source/.res/text/encrypt_binlog.removing.txt (hunk 1 above)
Comment (hrvojem) [reply doc/source/.res/text/encrypt_binlog.removing.txt:2]: 8.0.15-5 (:rn: tag?)

**Review by hrvojem — APPROVED** [review]
LGTM

**Pull request merged**